wallet/src/de/schildbach/wallet/SellDashPref.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2014-2015 the original author or authors.
```

Comment:
what is this?

Reply:
@rgenito This SellDashPref.java file which is used in SellDashFragment.java files  which is part of the WallofCoins module.

Reply:
@rgenito I had move those  file to WallOfCoin module.